tests/actions/PolicyMemberTest.ts

```diff
@@ -263,5 +264,133 @@ describe('actions/PolicyMember', () => {
             });
             await mockFetch?.resume?.();
         });
+
+        it('Add new members with admin/auditor role to the #admins room', async () => {
+            // Given a policy and an #admins room
+            const policyID = '1';
+            const adminRoomID = '1';
+            const defaultApprover = 'approver@gmail.com';
+            const ownerAccountID = 1;
+            const adminAccountID = 1234;
+            const adminEmail = 'admin@example.com';
+            const auditorAccountID = 1235;
+            const auditorEmail = 'auditor@example.com';
+            const userAccountID = 1236;
+            const userEmail = 'user@example.com';
+
+            await Onyx.set(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {
+                ...createRandomPolicy(Number(policyID)),
+                approver: defaultApprover,
+            });
+            await Onyx.set(`${ONYXKEYS.COLLECTION.REPORT}${adminRoomID}`, {
+                ...createRandomReport(Number(adminRoomID)),
+                policyID,
+                chatType: CONST.REPORT.CHAT_TYPE.POLICY_ADMINS,
+                participants: {
+                    [ownerAccountID]: {notificationPreference: 'always'},
+                },
+            });
+
+            // When adding a new admin, auditor, and user members
+            Member.addMembersToWorkspace({[adminEmail]: adminAccountID}, 'Welcome', policyID, [], CONST.POLICY.ROLE.ADMIN);
+            Member.addMembersToWorkspace({[auditorEmail]: auditorAccountID}, 'Welcome', policyID, [], CONST.POLICY.ROLE.AUDITOR);
+            Member.addMembersToWorkspace({[userEmail]: userAccountID}, 'Welcome', policyID, [], CONST.POLICY.ROLE.USER);
+
+            await waitForBatchedUpdates();
+
+            // Then only the admin and auditor should be added to the #admins room
+            const adminRoom = await new Promise<OnyxEntry<Report>>((resolve) => {
+                const connection = Onyx.connect({
+                    key: `${ONYXKEYS.COLLECTION.REPORT}${adminRoomID}`,
+                    callback: (report) => {
+                        Onyx.disconnect(connection);
+                        resolve(report);
+                    },
+                });
+            });
+            expect(adminRoom?.participants?.[adminAccountID]).toBeTruthy();
+            expect(adminRoom?.participants?.[auditorAccountID]).toBeTruthy();
+            expect(adminRoom?.participants?.[userAccountID]).toBeUndefined();
+        });
+    });
+
+    describe('removeMembers', () => {
+        it('Remove members with admin/auditor role from the #admins room', async () => {
```

Comment:
Can you extend the test to verify the report participants

Reply:
The remove onyx data doesn't change the participants.
https://github.com/Expensify/App/blob/ee29bcca041fce5eee2fc161f4d5af8c44afd648/src/libs/actions/Policy/Member.ts#L254-L304